src/__test__/components/data-exploration/generic-gene-table/CreateCellSetModal.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import CreateCellSetModal from 'components/data-exploration/generic-gene-table/CreateCellSetModal';
+
+describe('Create cell set modal', () => {
+  it('renders correctly', () => {
```

Comment:
what about a test that gets closed on cancel or on clicking the cross?

Reply:
There is a test in SelectionActions